dev/bots/analyze.dart

```diff
@@ -605,6 +607,7 @@ Future<void> verifyNoTrailingSpaces(String workingDirectory, { int minimumMatche
     .where((File file) => path.extension(file.path) != '.jpg')
     .where((File file) => path.extension(file.path) != '.ico')
     .where((File file) => path.extension(file.path) != '.jar')
+    .where((File file) => path.extension(file.path) != '.swp')
```

Comment:
do we need to validate swap files?

Reply:
This keep the analysis from getting flagged while I have vim open, otherwise I seemingly cant run them concurrently